bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,199 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+)[:]*(?P<column>\d+)*\s*'
+                     r'(?P<severity>error|warning)\s*(?P<message>.+)')
+class StylintBear:
+    """
+    Attempts to catch little mistakes (duplication of rules for instance) and
+    to enforce a code style guide. This is particularly important with Stylus,
+    which is unopinionated when it comes to syntax. Like Stylus itself, this
+    linter opts for flexibility over rigidity.
+    """
+
+    LANGUAGES = {'Stylus'}
+    REQUIREMENTS = {NpmRequirement('stylint', '1.5.9')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax', 'Redundancy'}
+    SEE_MORE = 'https://github.com/SimenB/stylint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        allow_block_keyword: bool=False,
+                        allow_brackets: str='never',
+                        allow_colons: str='always',
+                        allow_colors: str='always',
+                        allow_spaces_after_commas: str='always',
+                        allow_spaces_after_comments: str='always',
+                        allow_css_literal: str='never',
+                        max_selector_depth: bool=False,
+                        check_duplicates: bool=True,
+                        allow_efficient_properties: str='always',
+                        set_extend_preference: bool=False,
+                        indent_size: bool=False,
+                        allow_leading_zero: str='never',
+                        set_max_errors: bool=False,
+                        set_max_warnings: bool=False,
+                        mixed_spaces_and_tabs: bool=False,
+                        set_naming_convention: bool=False,
```

Comment:
isnt this https://github.com/SimenB/stylint#namingconvention--default-false-false--lowercase-dash--lowercase_underscore--camelcase--bem- ?  that isnt a `bool`